File: tests/fixtures/memcache-go/basic/no_imports.go
package main

// File without gomemcache import — must produce zero emits.

type fakeStore struct{}

func (f *fakeStore) Get(_ string) (interface{}, error) { return nil, nil }

func local() {
	s := &fakeStore{}
	s.Get("not-memcache")
}

File: tests/fixtures/memcache-rs/basic/src/no_imports.rs
// File without memcache crate use — must produce zero emits.

pub struct FakeStore;

impl FakeStore {
    pub fn get(&self, _key: &str) {}
    pub fn set(&self, _key: &str, _v: &str, _ttl: u32) {}
}

pub fn local(s: &FakeStore) {
    s.get("not-memcache");
    s.set("not-memcache", "val", 0);
}
